src/main/java/org/toradocu/translator/ConditionTranslator.java

```diff
@@ -591,27 +552,190 @@ private static void processTag(Tag tag, DocumentedMethod method) {
+   *
+   * @param method the DocumentedMethod
+   * @param comment the String comment to translate
+   * @param predicateSplitPoint index of the "if"
+   * @return the translation computed
+   */
+  private static String returnStandardPattern(
+      DocumentedMethod method, String comment, int predicateSplitPoint) {
+    String translation = "";
+    if (comment.contains(";")) comment = comment.replace(";", ",");
+    String predicate = comment.substring(0, predicateSplitPoint);
+    final String[] tokens = comment.substring(predicateSplitPoint + 3).split(",", 2);
+    String trueCase = tokens[0];
+    String falseCase = tokens.length > 1 ? tokens[1] : "";
+
+    if (!predicate.isEmpty() && !trueCase.isEmpty()) {
+      //          try {
+      String predicateTranslation = translateFirstPart(predicate, method);
+      if (predicateTranslation != null) {
+        String conditionTranslation = translateSecondPart(trueCase, method);
+
+        if (!predicateTranslation.isEmpty() && !conditionTranslation.isEmpty()) {
+          translation = conditionTranslation + " ? " + predicateTranslation;
+          // Else case might not be present.
+          String elsePredicate = translateLastPart(falseCase, method);
+          if (elsePredicate != null) {
+            translation = translation + " : " + elsePredicate;
+          }
+        }
+      }
+      /* To validate the return values of the method under test we will have a code similar
+       * to this in the generated aspect:
+       *
+       * if ( condition ) {
+       *   return check;
+       * else {
+       *   return anotherCheck;
+       * }
+       *
+       * Example: @return true if the values are equal.
+       * translation: args[0].equals(args[1]) ? true
+       *
+       * if (args[0].equals(args[1])) {
+       *   return result.equals(true);
+       * }
+       *
+       * We use the characters '?' and ':' to separate the different parts of the translation.
+       * We need that because the translation is currently a plain a String.
+       */
+      //          }
+      //          catch (IllegalArgumentException e) {
+      //            //TODO: Change the exception with one more meaningful.
+      //            // Pattern not supported.
+      //            log.warn("Unable to translate: \"" + comment + "\"");
+      //          }
+
+      else {
+        String match = lastAttemptMatch(method, comment);
+        if (match != null) translation = match;
+        else translation = "";
+      }
+    }
+    return translation;
+  }
+
+  /**
+   * Replace some common expressions in the comment with other standard easier to translate
+   * correctly.
+   *
+   * @param comment the String comment to sanitize
+   * @param method the DocumentedMethod
+   * @return the normalized comment
+   */
+  private static String normalizeComment(String comment, DocumentedMethod method) {
+    if (comment.contains("if and only if")) comment = comment.replace("if and only if", "if");
```

Comment:
`Iff` and `if` are not equivalent. For example `@return true iff the string is empty` should be translated as `args[0].isEmpty()==true ? result == true : result == false`